src/fuzzy_decisions/FuzzyLogicFunctions/Table.js

```diff
@@ -0,0 +1,25 @@
+import BaseFuzzyFunction from "./BaseFuzzyFunction";
+
+/**
+ * Note:
+ * Table assumes equal value within all the interval.
+ * For right trapezoid (e.g. non-equal lBorder.y and rBorder.y) create separate class
+ */
+class Table extends BaseFuzzyFunction {
+	constructor(args) {
```

Comment:
I root for named params for better readability

Reply:
Here we could use destructuring, but in case we'll extract some common code into the BaseFuzzyFunction it's kept this way.